src/Polly.Core/Simmy/Outcomes/ChaosOutcomeStrategyOptions.cs

```diff
@@ -28,5 +28,5 @@ public class ChaosOutcomeStrategyOptions<TResult> : ChaosStrategyOptions
     /// Defaults to <see langword="null"/>. This property is required.
     /// </value>
     [Required]
-    public Func<OutcomeGeneratorArguments, ValueTask<Outcome<TResult>?>> OutcomeGenerator { get; set; } = default!;
+    public Func<OutcomeGeneratorArguments, ValueTask<Outcome<TResult>?>>? OutcomeGenerator { get; set; }
```

Comment:
So just to double-triple check, as this is a nullable annotation on a reference type (`Func<T>`), this just changes nullable attributes and isn't binary breaking, right?

It's just source-breaking if you use `WarningsAsErrors` and use the getter and don't check for null?

Reply:
Correct, not a binary breaking change.

> It's just source-breaking if you use WarningsAsErrors and use the getter and don't check for null? 
Also correct.